temporal/api/cloud/identity/v1/message.proto

```diff
@@ -130,3 +130,40 @@ message ServiceAccountSpec {
     string description = 3;
 }
 
+
+message ApiKey {
+    // The id of the API Key
+    string id = 1;
+    // The current version of the API key specification
+    // The next update operation will have to include this version
+    string resource_version = 2;
+    // The API key specification
+    ApiKeySpec spec = 3;
+    // The current state of the API key
+    // Possible values: activating, activationfailed, active, updating, updatefailed, deleting, deletefailed, deleted, suspending, suspendfailed, suspended.
+    // For any failed state, reach out to Temporal Cloud support for remediation.
+    string state = 4;
+    // The id of the async operation that is creating/updating/deleting the API key, if any
+    string async_operation_id = 5;
+    // The date and time when the API key was created
+    google.protobuf.Timestamp created_time = 6;
+    // The date and time when the API key was last modified
+    // Will not be set if the API key has never been modified.
+    google.protobuf.Timestamp last_modified_time = 7;
+}
+
+message ApiKeySpec {
+    // The id of the owner to create the API key for
+    string owner_id = 1;
```

Comment:
Also we should mention that the id here will be the id of the service account when the type is service account.

Reply:
added comments.